proxy/http2/HTTP2.cc

```diff
@@ -727,7 +727,7 @@ http2_decode_header_blocks(HTTPHdr *hdr, const uint8_t *buf_start, const uint8_t
 }
 
 // Initialize this subsystem with librecords configs (for now)
-uint32_t Http2::max_concurrent_streams = 100;
+uint32_t Http2::max_concurrent_streams = 4294967295;
```

Comment:
This variable is override by `proxy.config.http2.max_concurrent_streams_in` in [Http2::init()](https://github.com/apache/trafficserver/blob/master/proxy/http2/HTTP2.cc#L743).
IMO, where we should change is `HTTP2_MAX_CONCURRENT_STREAMS` in [HTTP2.h#L57](https://github.com/apache/trafficserver/blob/master/proxy/http2/HTTP2.h#L57)